lib/pronto/config.rb

```diff
@@ -11,6 +11,10 @@ def initialize(config_hash = ConfigFile.new.to_h)
       end
     end
 
+    def consolidate_comments?
+      !@config_hash['consolidate_comments'].nil?
```

Comment:
I think we should consider changing that.

Meanwhile https://github.com/mmozuras/pronto#configuration should be updated I guess, for both, mentioning that there are no overrides as well as this new option.